easybuild/framework/easyblock.py

```diff
@@ -1445,7 +1445,20 @@ def extensions_step(self, fetch=False):
 
     def package_step(self):
```

Comment:
Hoping to help with this

Looking at this idea, is there an interface to pass the config state to such a function (or at least an example in the current code?)

I want to pass a lot of the EasyBlock object into package_fpm, but it seems a bit overkill to pass the whole thing...